queries.yaml

```diff
@@ -293,15 +295,15 @@ layers:
     transform:
       - vectordatasource.transform.tags_create_dict
       - vectordatasource.transform.tags_name_i18n
-      - vectordatasource.transform.admin_level_alternate_viewpoint
-      - vectordatasource.transform.tags_remove
       - vectordatasource.transform.admin_level_as_int
       - vectordatasource.transform.add_id_to_properties
       - vectordatasource.transform.detect_osm_relation
       - vectordatasource.transform.remove_feature_id
       - vectordatasource.transform.truncate_min_zoom_to_1dp
       - vectordatasource.transform.remap_viewpoint_kinds
       - vectordatasource.transform.unpack_viewpoint_claims
+      - vectordatasource.transform.admin_level_alternate_viewpoint
```

Comment:
Add comment that order is important and these ones need to be last in the list.

Reply:
fixed in 9506b97782500824588c5745c04dc520b202da17